test/test_unit_keystore_adapter.py

```diff
@@ -37,3 +35,10 @@ def test_is_keyring_accessible_return_true_when_get_password(self, mock_keyring)
     @patch.object(keyring, "get_password", side_effect=keyring.errors.KeyringError)
     def test_is_keyring_accessible_returns_false_on_keyring_error(self, mock_keyring):
         self.assertEquals(False, KeystoreAdapter.is_keyring_accessible())
+
+    @patch.object(util_functions, "configure_env_var_with_keyring")
+    def test_env_variables_set_on_keyring_adapter_import(self, mock_utils):
+        import importlib
+        import conjur.wrapper.keystore_adapter
+        importlib.reload(conjur.wrapper.keystore_adapter)
+        mock_utils.assert_called_once()
```

Comment:
the name of the unit test is `test_env_variables_set_on_keyring_adapter_import` but we just set it is called and not that anything was set

Reply:
what name do you think we should use?

Reply:
can we check that the env has the environment var?